projects/lib/src/oauth-service.ts

```diff
@@ -604,7 +613,7 @@ export class OAuthService extends AuthConfig implements OnDestroy {
         }
         if (!this.validateUrlForHttps(this.userinfoEndpoint)) {
             throw new Error(
-                'userinfoEndpoint must use https, or config value for property requireHttps must allow http'
+                'userinfoEndpoint must use HTTP or config value for property requireHttps must allow HTTP.'
```

Comment:
Same comment as for the other error message, we should either change it back or use some form of updated/improved text indicating normally ssl/tls is required.